File: executable/r/error-bar-asymmetric-array.r
library(plotly)
p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  x = c(1, 2, 3, 4), 
  y = c(2, 1, 3, 4), 
  error_y = list(
    array = c(0.1, 0.2, 0.1, 0.1), 
    type = "data", 
    symmetric = FALSE, 
    arrayminus = c(0.2, 0.4, 1, 0.2)
  ), 
  type = "scatter"
)


response <- p$plotly(trace0, kwargs=list(filename="error-bar-asymmetric-array", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/julia/error-bar-asymmetric-array.jl
using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

data = [
  [
    "x" => [1, 2, 3, 4], 
    "y" => [2, 1, 3, 4], 
    "error_y" => [
      "array" => [0.1, 0.2, 0.1, 0.1], 
      "type" => "data", 
      "symmetric" => false, 
      "arrayminus" => [0.2, 0.4, 1, 0.2]
    ], 
    "type" => "scatter"
  ]
]

response = Plotly.plot([data], ["filename" => "error-bar-asymmetric-array", "fileopt" => "overwrite"])
plot_url = response["url"]